source/dotnet/Library/AdaptiveCards.Rendering.Wpf/RendererUtil.cs

```diff
@@ -11,6 +12,32 @@ namespace AdaptiveCards.Rendering.Wpf
 {
     static class RendererUtil
     {
+        /// <summary>
+        /// Checks if any of the fonts to be used for the TexBlock is installed, and if so, returns it
+        /// </summary>
+        /// <param name="fontList">Font list to check if any exist</param>
+        /// <returns>First font in list that is installed, otherwise, first element</returns>
+        public static string GetFontFamilyFromList(string fontList)
+        {
+            var installedFonts = new InstalledFontCollection();
+            string[] fontFamilies = fontList.Split(',');
+
+            for (int i = 0; i < fontFamilies.Length; ++i)
+            {
+                string fontFamily = fontFamilies[i].Trim('\'');
+
+                foreach (var installedFontFamily in installedFonts.Families)
+                {
+                    if (installedFontFamily.Name == fontFamily)
+                    {
+                        return fontFamily;
+                    }
+                }
+            }
+
+            return fontFamilies[0].Trim();
```

Comment:
this doesn't make sense, if there is not matching font amonst the installed fonts, then why do we return a first font from the list? Shouldn't we return a font that's existing?